bika/lims/browser/bika_listing.py

```diff
@@ -453,7 +453,7 @@ class BikaListingView(BrowserView):
 
     # Additional indexes to be searched
     # any index name not specified in self.columns[] can be added here.
-    filter_indexes = ['Title', 'Description', 'SearchableText']
+    filter_indexes = ['title', 'SearchableText']
```

Comment:
Good catch. `Title` and `Description` are not defined as indexes neither in `bika_catalog_analysisrequest_listing` https://github.com/senaite/bika.lims/blob/senaite-integration/bika/lims/catalog/analysisrequest_catalog.py#L18-L38 nor in https://github.com/senaite/bika.lims/blob/senaite-integration/bika/lims/catalog/catalog_basic_template.py#L8-L31

Nevertheless, all lists inherits from bika_listing and, depending on the type of objects to list, they fetch data from different catalogs and we cannot assure all them have the indexes `title` and `SearchableText`. For this reason, **we need to take special care to override `filter_indexes` variable correctly in child classes**.